events/tests/test_api.py

```diff
@@ -229,3 +232,217 @@ def test_event_list_with_publisher_filters(self):
             self.assertEqual(response.status_code, status.HTTP_200_OK)
             # note that org-2 is replaced by org-1
             self.assertEqual(len(response.data['data']), 2)  # event-1 and event-2
+
+    def test_random_user_create_event_denied(self):
```

Comment:
I guess for completeness' sake, we should add the same tests for a random user trying to PUT or DELETE any events?